spec/keri.md

```diff
@@ -1,61 +1,59 @@
 ## KERI Fundamentals
 
-[[ref: KERI]] ([[ref: Key Event Receipt Infrastructure]]) is a methodology for managing cryptographic keys, plus the identifiers and verifiable data structures that depend on them. KERI was first described in an [academic paper](https://arxiv.org/abs/1907.02143), and is now a [draft IETF RFC](https://weboftrust.github.io/ietf-keri/draft-ssmith-keri.html). The open source community that develops KERI-related technologies can be found at https://github.com/WebOfTrust.
+[[ref: Key Event Receipt Infrastructure (KERI)]] is a protocol for managing cryptographic keys, identifiers, and associated verifiable data structures. KERI was first described in an [academic paper](https://arxiv.org/abs/1907.02143), and its [specification](https://github.com/trustoverip/tswg-keri-specification) is currently incubated under [Trust Over IP Foundation](https://trustoverip.org/). The open source community that develops KERI-related technologies can be found at https://github.com/WebOfTrust. This section outlines the fundamentals and components of the KERI protocol that are related to the `did:webs` method.
 
-[[def: Autonomic Identifier, AID]]
+### Autonomic Identifier (AID)
 
-A globally unique string that is bound to cryptographic keys in a special way. AIDs and the associated structures, the [[ref: KEL]] and [[ref: TEL]], have most of the properties that DIDs require, plus a few that give them unusually special security and decentralization.
+An [[ref: autonomic identifier (AID)]] is a globally-unique persistent self-certifying identifier that serves as the primary root-of-trust of the KERI protocol. An AID is cryptographically bound to a [[ref: KEL]] that determines the evolution of its [[ref: key state]] using the [[ref: pre-rotation]] mechanism. AIDs and the underlying KERI protocol, by themselves, satisfy most of the [properties](https://www.w3.org/TR/did-core/#design-goals) that DIDs require, including decentralization, control, security, proof-based, and portability. For example, DIDs that have the same AID component are considered [equivalent](#equivalent-identifiers), allowing AIDs to be portable across different DID methods such as `did:webs` and `did:keri`.
 
-[[def: Key Event Log, KEL]]
+### Key Event Log (KEL)
 
-The binding between an AID and its cryptographic keys is proved by a data structure called a key event log (KEL). This data structure has certain blockchain-like properties. However, a KEL differs from blockchain technology in at least two important ways:
+The binding between an [[ref: AID]] and its cryptographic keys is proved by a data structure called a [[ref: key event log (KEL)]] that allows the [[ref: key states]] of the AID to evolve. For a `did:webs` DID, a KEL is an essential component in the [[ref: KERI event stream]] that is used to verify its authenticity.
```

Comment:
Same comment about the definition being in terminology (see above) so this section shouldn't have the first sentence.

Reply:
Pushing to a future convo/issue/PR